src/client/app/app.component.ts

```diff
@@ -1,11 +1,9 @@
 import { Component } from '@angular/core';
 import { ROUTER_DIRECTIVES, Routes } from '@angular/router';
 
-import { AboutComponent } from '../+about/index';
-import { HomeComponent } from '../+home/index';
-import { NameListService } from '../shared/index';
-import { NavbarComponent } from './navbar.component';
-import { ToolbarComponent } from './toolbar.component';
+import { AboutComponent } from './+about';
```

Comment:
As of the Styleguide the barrel syntax (Style 04-10 Create and Import Barrels - https://angular.io/styleguide#!#application-structure )for the imports list the following example:

```
import {
  CONFIG,
  EntityService,
  ExceptionService,
  FilterTextComponent,
  InitCapsPipe,
  SpinnerService,
  ToastService
} from '../shared';
```

Within the folder `shared` there is an `index.ts` barrel file. However, if i import it like this, i get an exception at runtime:

![screenshot 2016-05-11 17 49 08](https://cloud.githubusercontent.com/assets/1188033/15187267/c2d30d26-17a0-11e6-897f-09e2f5e40789.png)

With this syntax, a lookup is made for +about.js, +home.js, shared.js; yet the corresponding files are +about/index.js, +home/index.js and shared/index.js.

I dont know if that is a problem of the SystemJS Configuration.

Any help on this would be appreciated.